dotnet/dotnet-guestbook/.readmes/vscode/README.md

```diff
@@ -31,44 +31,52 @@ The Guestbook sample demonstrates how to deploy a Kubernetes application with a
 ---
 ## Getting Started
 
-### Run the app locally with minikube
+### Set up a Kubernetes cluster
 
-1. To run your application, click on the Cloud Code status bar and select ‘Run on Kubernetes’.  
-![image](./img/status-bar.png)
+#### Minikube
+ 
+ Cloud Code uses [minikube](https://minikube.sigs.k8s.io/docs/) to create a free local cluster.
 
-2. Select ‘Run locally using minikube’ when prompted. Cloud Code runs your app in a local [minikube](https://minikube.sigs.k8s.io/docs/start/) cluster.  
-![image](./img/create-k8s-cluster.png)
+1. Navigate to the **Cloud Code - Kubernetes** explorer using the left side taskbar.
```

Comment:
We changed the name of the Kubernetes explorer to Clusters explorer.

To be technically correct, we should say
"Navigate to the **Clusters explorer** in the **Cloud Code - Kubernetes** sidebar".

Reply:
the left side bar is actually called Activity bar in vscode terminology.

Maybe we should use the below?
** using the left side Activity bar

Reply:
Ah, thanks! How about "Navigate to the **Clusters explorer** in the Cloud Code - Kubernetes View using the left side Activity bar."